manifests/l/LizardByte/Sunshine/0.20.0/LizardByte.Sunshine.installer.yaml

```diff
@@ -1,15 +1,15 @@
-# Created with WinGet Releaser v2 using Komac v1.6.0
+# Created with WinGet Releaser v2 using Komac v1.9.1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: LizardByte.Sunshine
 PackageVersion: 0.20.0
 InstallerType: nullsoft
 Scope: machine
 UpgradeBehavior: install
-ReleaseDate: 2023-05-29
+ReleaseDate: 2023-07-22
 Installers:
 - Architecture: x64
-  InstallerUrl: https://github.com/LizardByte/Sunshine/releases/download/v0.20.0/sunshine-windows-installer.exe
-  InstallerSha256: 95D22F07256798C2C1F32E813E9EE2FC2888428FFC431B92D6C0CE07C80E2257
+  InstallerUrl: https://github.com/utkarshdalal/Sunshine-GameAway/releases/download/v0.20.0/sunshine-windows-installer.exe

```

Comment:
Why the repository should change to something rather than the official one?

Reply:
They forked our repo, added the required secrets, and ran our workflows.